testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -244,17 +245,24 @@ public void InvokeFromMethodActionRpc(Action methodInfo)
     }
 
     [ClientRpc]
-    public void TriggerActionIdClientRpc(string actionId, byte[] args, ClientRpcParams clientRpcParams = default)
+    public void TriggerActionIdClientRpc(string actionId, byte[] args, bool ignorException, ClientRpcParams clientRpcParams = default)
     {
-        Debug.Log($"received RPC from server, client side triggering action ID {actionId}");
+        BaseMultiprocessTests.MultiProcessLog($"received RPC from server, client side triggering action ID {actionId}");
         try
         {
             ExecuteStepInContext.AllActions[actionId].Invoke(args);
         }
         catch (Exception e)
         {
             WriteErrorServerRpc(e.Message);
-            throw;
+            if (!ignorException)
+            {
+                throw;
+            }
+            else
+            {
+                Instance.ClientFinishedServerRpc();
```

Comment:
mmm so this is duplicating logic from step's Invoke, however, it ignores the logic about if (!m_WaitMultipleUpdates)
I'd forward ignoreException to invoke and move your if (!ignoreException) check to Invoke? what do you think?

Reply:
NO!
**Leave this here.**  It is the client side handling exceptions when you are explicitly testing for exceptions.
Otherwise, the client will have to timeout before it is considered "complete".  
This prevents that from happening.